lib/ansiblelint/formatters/junitxml.py

```diff
@@ -0,0 +1,58 @@
+r"""JUnit XML formatter
+
+The XML Schema of JUnit XML format:
+
+- https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd
+  [http://windyroad.com.au/2011/02/07/apache-ant-junit-xml-schema/]
+"""
+from __future__ import absolute_import
+import datetime
+
+from .base import BaseFormatter
+from ..version import __version__ as VERSION
```

Comment:
Add this boilerplate for defaulting to new-style classes automatically:
```suggestion
from ..version import __version__ as VERSION


__metadata__ = type
```